### PR TITLE
Portfolio: setup and mobile first

### PR DESCRIPTION
**Pull request by Bupilipili**
In this branch.

-Updated the style.css file
-Updated the index.html file

### CODE REVIEW

**Review by wayungi — COMMENTED** [review]
Hi <STUDENT_MENTION>, 
image

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ 

- [ ] You have submitted a project that is invalid. please remove all instructions & comments from your readme. Clue: they begin with `>`

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit.

File: index.html [hunk 1]
```diff
@@ -36,5 +36,374 @@ <h1 id="head1">Hey there.<br> I’m Erick.<br>
...
      </main>
+     <section>
+        <div id="content2"><p class="works">My Recent Works</p>
+           <img id="img2" src="./features/images/Indicator.png">
+        </div>
+         <div class="main-container">
+            <div class="grid-container">
+              <div class="item1"></div>
+               <div class="flex">
+                <div class="item2">Multi-Post Stories Gain+Glory</div>
```
Comment (Strangeal) [inline index.html]: - [x] Kindly consider using a header tag `hx` eg `h2, h3, h4, h5, h6` for the title of each card as it's semantically correct and also helps make your website accessible and SEO friendly.

File: index.html [hunk 2]
```diff
@@ -36,5 +36,374 @@ <h1 id="head1">Hey there.<br> I’m Erick.<br>
...
+                    <li class="item3">JavaScript</li>
+                    <li class="item3">HTML</li>
+                  </ul>
+                  <a href="#" target="_blank" class="item7">See project</a>
+                </div>
+              </div>
+              <div class="grid-container">
+                <div class="item1"></div>
+                 <div class="flex">
+                  <div class="item2">Multi-Post Stories Gain+Glory</div>
```
Comment (Strangeal) [inline index.html]: - [x] Kindly consider using a header tag `hx` eg `h2, h3, h4, h5, h6` for the title of each card as it's semantically correct and also helps make your website accessible and SEO friendly.

File: index.html [hunk 3]
```diff
@@ -36,5 +36,374 @@ <h1 id="head1">Hey there.<br> I’m Erick.<br>
...
+                   <li class="item3">JavaScript</li>
+                   <li class="item3">HTML</li>
+                 </ul>
+                  <a href="#" target="_blank" class="item7">See project</a>
+                 </div>
+               </div>
+               <div class="grid-container">
+                 <div class="item1"></div>
+                  <div class="flex">
+                   <div class="item2">Multi-Post Stories Gain+Glory</div>
```
Comment (Strangeal) [inline index.html]: - [x] Kindly consider using a header tag `hx` eg `h2, h3, h4, h5, h6` for the title of each card as it's semantically correct and also helps make your website accessible and SEO friendly.

File: index.html [hunk 4]
```diff
@@ -36,5 +36,374 @@ <h1 id="head1">Hey there.<br> I’m Erick.<br>
...
+                     <li class="item3">JavaScript</li>
+                     <li class="item3">HTML</li>
+                   </ul>
+                   <a href="#" target="_blank" class="item7">See project</a>
+                 </div>
+               </div>
+               <div class="grid-container">
+                 <div class="item1"></div>
+                  <div class="flex">
+                   <div class="item2">Multi-Post Stories Gain+Glory</div>
```
Comment (Strangeal) [inline index.html]: - [x] Kindly consider using a header tag `hx` eg `h2, h3, h4, h5, h6` for the title of each card as it's semantically correct and also helps make your website accessible and SEO friendly.

File: index.html [hunk 5]
```diff
@@ -36,5 +36,374 @@ <h1 id="head1">Hey there.<br> I’m Erick.<br>
...
```
Comment (Strangeal) [inline index.html]: - [x] Kindly consider using a header tag `hx` eg `h2, h3, h4, h5, h6` for the title of each card as it's semantically correct and also helps make your website accessible and SEO friendly.

File: index.html [hunk 6]
```diff
@@ -36,5 +36,374 @@ <h1 id="head1">Hey there.<br> I’m Erick.<br>
...
```
Comment (Strangeal) [inline index.html]: - [x] Kindly consider using a header tag `hx` eg `h2, h3, h4, h5, h6` for the title of each card as it's semantically correct and also helps make your website accessible and SEO friendly.

File: index.html [hunk 7]
```diff
@@ -36,5 +36,374 @@ <h1 id="head1">Hey there.<br> I’m Erick.<br>
            <img id="img" src="./features/images/Disabled.png">
       </section>
      </main>
+     <section>
+        <div id="content2"><p class="works">My Recent Works</p>
```
Comment (Strangeal) [inline index.html]: - [x] Kindly consider using a header tag `hx` eg `h2, h3, h4, h5, h6` for the title of each card as it's semantically correct and also helps make your website accessible and SEO friendly.

File: index.html [hunk 8]
```diff
@@ -36,5 +36,374 @@ <h1 id="head1">Hey there.<br> I’m Erick.<br>
...
+                     <li class="item3">HTML</li>
+                   </ul>
+                  <a href="#" target="_blank" class="item7">See project</a>
+                 </div>
+               </div>
+            </div>
+ </section>
+   <section>
+    <div id="About-me">
+       <p class="about">About me</p>
```
Comment (Strangeal) [inline index.html]: - [x] Kindly consider using a header tag `hx` eg `(h2, h3, h4, h5, h6)` for the title of each card as it's semantically correct and also helps make your website accessible and SEO friendly.

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @Bupilipili, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

## Highlights
Correct use of GitHub Flow ✅
Well-structured files ✅
All linter checks are passing ✅
Descriptive PR title with a great summary ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 7 above)
Comment (tufoinnkuo10) [inline index.html]: - [x] please, maybe you forgot. Like the previous reviewer noted, kindly make the `my recent works` title an `hx`

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Bupilipili 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally


Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by emmiiorji — APPROVED** [review]
Hi @Bupilipili, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>


## Good Highlights✔️

- Requested change has been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**